### PR TITLE
2128/alt contact street field 2

### PR DESCRIPTION
**Pull request by adriencyberspace**
# Pull Request Template

## Issue Overview

This PR addresses #2128 

- [x] This change addresses the issue in full
- [ ] This change addresses only certain aspects of the issue
- [ ] This change is a dependency for another issue
- [ ] This change has a dependency from another issue

## Description

This PR adds a second street field for unit / apt no. for alternate contact on the user form.

## How Can This Be Tested/Reviewed?

Entry Point: http://localhost:3000/applications/contact/alternate-contact-contact

1. Begin an application and select that you have an alternate contact
2. Fill out alternate contact's address and use the second street field (Apt or Unit #)
3. At the review section of the application, you will see your alt contact's full address
4. In partners, open your application and you will also see your alt contact's full address

image

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have added QA notes to the issue with applicable URLs
- [x] I have performed a self-review of my own code
- [x] I have reviewed the changes in a desktop view
- [x] I have reviewed the changes in a mobile view
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules
- [x] I have assigned reviewers
- [ ] I have run `yarn generate:client` and/or created a migration if I made backend changes that require them
- [ ] I have exported any new pieces added to ui-components
- [x] My commit message(s) is/are polished, and any breaking changes are indicated in the message and are well-described
- [ ] Commits made across packages purposefully have the same commit message/version change, else are separated into different commits

## Reviewer Notes:

Steps to review a PR:

- Read and understand the issue, and ensure the author has added QA notes
- Review the code itself from a style point of view
- Pull the changes down locally and test that the acceptance criteria is met
- Also review the acceptance criteria on the Netlify deploy preview (noting that these do not yet include any backend changes made in the PR)
- Either explicitly ask a clarifying question, request changes, or approve the PR if there are small remaining changes but the PR is otherwise good to go

## On Merge:

If you have one commit and message, squash. If you need each message to be applied, rebase and merge.

### CODE REVIEW

File: ui-components/src/helpers/address.tsx [hunk 1]
```diff
@@ -40,6 +40,12 @@ export const MultiLineAddress = (props: AddressProps) => {
       )}
       <Markdown children={props.address.street || ""} />
       <br />
+      {props.address.street2 && (
```
Comment (emilyjablonski) [inline ui-components/src/helpers/address.tsx:43]: Address formats usually have street 2 on the same line, I actually just made this change in another PR I have up if you wanna take a look: https://github.com/bloom-housing/bloom/pull/2675/files#diff-850c89614d0470e86af5b2769705e03fad7e3869c92eb5eca4eb4025329b742aR1

**Review by emilyjablonski — CHANGES_REQUESTED** [review]
After the street2 multiline address formatting update, lgtm!

**Review by emilyjablonski — APPROVED** [review]
LGTM after the tests go green

**Review by YazeedLoonat — CHANGES_REQUESTED** [review]
Looks good, please update ui-component tests so they can pass

also in your qa notes the url should point to the dev site

**Pull request merged**